packages/react-scripts/config/webpack.config.js

```diff
@@ -599,19 +607,21 @@ module.exports = function(webpackEnv) {
       // Generate a service worker script that will precache, and keep up to date,
       // the HTML & assets that are part of the Webpack build.
       isEnvProduction &&
-        new WorkboxWebpackPlugin.GenerateSW({
-          clientsClaim: true,
-          exclude: [/\.map$/, /asset-manifest\.json$/],
-          importWorkboxFrom: 'cdn',
-          navigateFallback: publicUrl + '/index.html',
-          navigateFallbackBlacklist: [
-            // Exclude URLs starting with /_, as they're likely an API call
-            new RegExp('^/_'),
-            // Exclude URLs containing a dot, as they're likely a resource in
-            // public/ and not a SPA route
-            new RegExp('/[^/]+\\.[^/]+$'),
-          ],
-        }),
+        new WorkboxWebpackPlugin.GenerateSW(
```

Comment:
As a complement to this PR after release, I believe we should already think of support in using injectManifest instead of generateSW.

In scenarios where the cache of requests and other features makes sense, GenerateSW does not respond.

More details: https://developers.google.com/web/tools/workbox/modules/workbox-webpack-plugin#which_plugin_to_use

Should we think about this support now or after this release?